index.html

```diff
@@ -364,6 +365,24 @@ <h2>
       <h2>
         Algorithms
       </h2>
+      <section>
+        <h3>
+          Resize the title bar area
+        </h3>
+        <p>
+          To <dfn data-lt="title bar region resizes">resize the title bar
+          area</dfn> for a window |win|, run these steps:
+        </p>
+        <ol>
+          <li>If |win|'s [=overlay=] has had its width or height changed (e.g.
+          as a result of the user resizing the browser window, or changing the
+          page zoom factor, or extensions/other UI appearing or disappearing on
```

Comment:
mention of extensions here is very Chromium specific. maybe we should omit it? and just keep the "other UI appearing" part?

Reply:
removed extensions